File: C++ Programming/Competitive Programming/Coding Challenges 9/program5.cpp
// 5. Write a program which accept N and print first 5 multiples of N.
// Input : 4
// Output : 4 8 12 16 20

#include <iostream>
using namespace std;

void MultipleDisplay(int iNo)
{
    int i = 1;
    int iMultiple = iNo;

    if (iNo < 0)
    {
        iNo = -iNo;
        iMultiple = iNo;
    }

    while (i <= 5)
    {
        cout << iMultiple << "\t";
        iMultiple = iMultiple + iNo;
        i++;
    }
}

int main()
{
    int iValue = 0;

    cout << "Enter Number : ";
    cin >> iValue;

    MultipleDisplay(iValue);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 17/program5.java
/*
5.Write a program which accept number from user and count frequency of such a
digits which are less than 6.
Input : 2395
Output : 3
Input : 1018
Output : 3
Input : 9440
Output : 3
Input : 96672
Output : 1
*/

import java.util.Scanner;

class program5
{
    static int Count(int iNo)
    {
        int iCount = 0;

        String str = Integer.toString(iNo);

        for (int i = 0; i < str.length(); i++)
        {
            char ch = str.charAt(i);

            if (ch >= '0' && ch <= '5')
            {
                iCount++;
            }
        }

        return iCount;
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        int iRet = 0;

        System.out.print("Enter number : ");
        iValue = sobj.nextInt();

        iRet = Count(iValue);

        System.out.print("Output is : " + iRet);

        sobj.close();
    }
}